year-pages/2019-conferences-list.md

```diff
@@ -3,11 +3,13 @@
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [React India](https://www.reactindia.io/) | 22-24 Aug | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more | No |
+| [Agile India 2019](https://2019.agileindia.org/) | 17th - 23rd Mar | Sheraton Grand Bangalore Hotel, Bengaluru | Asia's Largest and Premier International conference on Leading Edge Software Development Methods | No |
```

Comment:
I can't find a CFP link for this conference (or any mention of it). This repo is only accepting conferences which have CFPs, so if you could link to a call for papers (a twitter link is also ok) I am happy to accept this one.